packages/auth/package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MPL-2.0",
   "private": true,
   "scripts": {
-    "start": "cross-env NODE_ENV=development nodemon --exec ts-node -r tsconfig-paths/register src/index.ts",
+    "start": "cross-env NODE_ENV=development NODE_OPTIONS=--dns-result-order=ipv4first  nodemon --exec ts-node -r tsconfig-paths/register src/index.ts",
```

Comment:
@rikukissa I think the main reason for Mac users encountering this issue is the change in DNS resolution strategy in Node 17 and afterward. 
https://github.com/nodejs/node/issues/40702#issuecomment-1103623246

Reply:
The fix seems to be working for me @tahmidrahman-dsi

data-seeder is still failing though, think it needs to be added there as well. Also, I didn't test, but would data-generator require it in Farajaland too? Farajaland otherwise works as well for me now

Reply:
Thanks very much @naftis 🙏🏼 I have added that var in the seed script https://github.com/opencrvs/opencrvs-core/pull/6439/commits/40e21a3488719ab8652543cd128cb6caaabc7f52 Now it should work

Reply:
```
➜  opencrvs-core git:(upgrade-node@18) ✗ yarn seed:dev
yarn run v1.22.19
$ ACTIVATE_USERS=true lerna run seed --stream --scope @opencrvs/data-seeder
lerna notice cli v5.6.2
lerna notice filter including "@opencrvs/data-seeder"
lerna info filter [ '@opencrvs/data-seeder' ]
lerna info Executing command in 1 package: "yarn run seed"
@opencrvs/data-seeder: $ NODE_OPTIONS=--dns-result-order=ipv4first ts-node -r tsconfig-paths/register src/index.ts
@opencrvs/data-seeder: Seeding roles
@opencrvs/data-seeder: [
@opencrvs/data-seeder:   {
@opencrvs/data-seeder:     message: 'FetchError: request to http://localhost:3030/getUser failed, reason: connect ECONNREFUSED ::1:3030',
@opencrvs/data-seeder:     locations: [ [Object] ],
@opencrvs/data-seeder:     path: [ 'getSystemRoles' ],
@opencrvs/data-seeder:     extensions: { code: 'UNAUTHENTICATED', exception: [Object] }
@opencrvs/data-seeder:   }
@opencrvs/data-seeder: ]
@opencrvs/data-seeder: error Command failed with exit code 1.
@opencrvs/data-seeder: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
➜  opencrvs-core git:(upgrade-node@18) ✗ node -v
v18.19.0
```
🤔 😕

Reply:
This might be something we should debug on Mac, if you don't have one :s

Reply:
I just came up with a way that can reproduce this Mac users' issue by updating my `/etc/hosts`.  I added an alias of `localhost` to the ipv6 `::1`. Now I can see this error